pi5/board.go

```diff
@@ -390,10 +393,10 @@ type pinctrlpi5 struct {
 
 	/* Custom PinCTRL Params Here: */
 	dtBaseNodePath string
-	mem_fd         int32
-	virt_addr      *uint64
-	phys_addr      *uint64
-	gpiomem_idx    [4]rune
+	memFD          int32
```

Comment:
can you put a comment next to all of these fields explaining what theyre used for?